weave-js/src/components/PagePanelComponents/Home/Browse3/pages/CallPage/DataTableView.tsx

```diff
@@ -392,7 +392,7 @@ export const DataTableView: FC<{
   const displayRows = 10;
   const hideFooter = USE_TABLE_FOR_ARRAYS && gridRows.length <= displayRows;
   const headerHeight = 40;
-  const footerHeight = 52;
+  const footerHeight = 40;
```

Comment:
what does this do?

Reply:
This is the height of the footer bar in px after the smaller button change. It gets passed to help calculate the height of the datatable above it.